src/pranitaPJun20/ArrayReverseNum.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Program 3: WAP to reverse number (Consider Input as Integer not as String).
+Example:-
+Input:- 12345
+Output:- 54321
+ */
+package pranitaPJun20;
+
+public class ArrayReverseNum {
+	// Method to find the reverse of the given number
+	void findReverseOfNumber(int number) {
+		int reverse = 0;
+		int input = number;
+		while (number > 0) {
+			int lastDigit = number % 10;
+			reverse = (reverse * 10) + lastDigit;
```

Comment:
No need to take extra variable. Can achieve same result by replacing equation of line 15 in line 16 for lastDigit.